src/cluster/managers/documentation/CommandDocumentationManager.ts

```diff
@@ -88,6 +88,31 @@ export class CommandDocumentationManager extends DocumentationTreeManager {
         };
     }
 
+    #listCommandNames(names: readonly string[]): string {
+        const charLimit = discord.getLimit('embed.field.value') - 6;
+        if (names.reduce((p, c) => p + c.length + 2, 0) - 2 <= charLimit)
+            return `\`\`\`${names.join(', ')}\`\`\``;
+
+        const remaining = [...names];
+        let missing = 0;
+        const postfix1 = '+ ';
+        const postfix2 = ' more';
+        function contentLength(): number {
+            return remaining.reduce((p, c) => p + c.length + 2, 0) - 2
+                + postfix1.length
+                + postfix2.length
+                + missing.toString().length;
+        }
+        while (contentLength() > charLimit) {
+            remaining.pop();
+            missing++;
+        }
+        if (remaining.length === 0)
+            return `${missing} ${pluralise(missing, 'command')}`;
```

Comment:
So if the whole content is still too large, you just send (for example) `15 commands` as the returned value?

Reply:
Correct, this is for the case where the first element is > 1024 characters long, and so we cannot show any inside the code block